data/hai/hai reveal 4 middle.txt

```diff
@@ -966,8 +966,30 @@ mission "Hai Reveal [A08] Symbol Identification"
 		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
 		
 		conversation
-			`After landing, you search up the way to the adjunct research library at the Navy Intelligence Observatory in Moscow. It takes three shuttle journeys and one high-speed rail trip, but you reach the building in under three hours. For someone of your background there are parts of this trip that seem dizzyingly unreal. As well traveled as you now are, actually navigating the urban environments of <origin> away from any particular spaceport is an experience not quite like any other. Once you reach the building in question you take a moment at the reception desk to interpret 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become. Some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
+			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your business of angry cat symbols.`
+			`	At an information desk, you inquire about the route to the library, and are given two options. You can take a very expensive private taxi service, which will be payed for by the Syndicate. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
+			choice
+				`	(Take the direct route.)`
+					goto "take the direct route"
+				`	(Enjoy the sights along the way.)`
+			`	Moscow is thousands of kilometers away, so you travel most of the route on shuttlecraft. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but it's too dull to be restrained to just one planet.`
+			`	You fly past mountain ranges, lakes, and deserts, and the occasional tundra. Everywhere, you see the gray cities of Earth packed in their ancient locations. Even from kilometers above, the cities' vast megabuildings are an imposing sight. There's still some farmland, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```

Comment:
Just correcting for the actual geography in between those places.
```suggestion
			`	You fly past mountain ranges, lakes, and tundra, both bare and forested. The gray cities of Earth stand out when you pass them; packed in their ancient locations and their new ones. Even from far above, the cities' vast megabuildings are an imposing sight. There's still some farmland on this route, even now that farming has largely moved off-world; but as you well know, all of that isn't enough for the twenty billion people living on Earth.`
```

Reply:
You would pass over the Karakorum Desert (Kazakhstan) if you landed at Ulaanbaatar. Mongolia is half desert itself.

Reply:
The Karakorum Desert is in Turkmenistan, not Kazakhstan, and you wouldn't pass over it on the way to Nizhny.
The recognisably desert part of Mongolia is also south of any vaguely direct route you would take from Ulaanbaatar.
While other areas might technically be "deserts", they're also more accurately arid tundras.
So no.

Reply:
I thought the player was going Ulaanbataar -> Moscow, not Nizhny?

Anyway regardless of the exact Karakorum desert positioning, most of Central Asia is desertic/arid (regardless of the exact classification) so it bears mentioning.

Reply:
You don't know how climate change and terraforming will change Earth between now and then. I was just putting some geographical features in the paragraph; I don't particularly care which ones.  (Other than "ocean" or "dozens of super volcanoes.") If you can convince MasterOfGrey it needs to be something else, I'll change it.

Reply:
If you take 3 shuttles like it says, you'd probably land or go over somewhere around Astana. From there, you would be able to see desert. There's a Karakum desert in Kazakhstan, but that one is covering the Aral Sea.

Reply:
From another comment chain, @bene-dictator pointed out that Tundras don't have forests (or to put it another way, if it has a forest, it is not a tundra).

I researched it and found this: "Tundra comes from the Finnish word tunturi, meaning treeless plain." There are a lot of informational sites on the flora and fauna of tundras, and although there may be scattered trees, the foliage is mostly quite low.

So even the word itself says it is treeless. No forested tundras, sorry.